guides/Starters/Vanilla_js.md

```diff
@@ -0,0 +1,19 @@
+## OVERVIEW:
+The term Vanilla in english is termed as "anything having no special or extra features".Vanilla javascript is basically plain old javascript,understandable by javascript compiler,which means writing DOM manipulation manually. It is fast,lightweight framework which enables one to build incredible and powerful javascript applications. It refers to the framework which makes use of javascript without any libraries but it sacrifices code legibility in order to achieve better performance.Facebook,Google,Youtube,Yahoo,Wikipedia,WindowsLive Twitter,Amazon,LinkedIn,MSN,eBay,Microsoft,Tumblr,Apple,Pinterest,PayPal,Reddit,Netflix,Stackoverflow makes use of vanilla js.
+
+#### WHEN TO USE VANILLA JS ?
+Vanilla is fine when one aims to build a content-heavy website.
+
+Use Vanilla when primary client side app technology(rendering html,routing,validation,data mapping,calling services and authentication) is different that javascript(.net,jsp etc) and one just want DOM manipulations to already rendered DOM without disturbing core mechanism of rendering.

```

Comment:
1) Kindly, change "that" to "than". 
2)  Didn't get this ``` already rendered DOM without``` ? Can you reframe this sentence, if possible! :smirk: